### DISCUSSION TITLE
Does reporter support html report?

### DISCUSSION THREAD

**Discussion by sgolyala1** [test-results-reporter/reporter#114] [Q&A]
Hi @ASaiAnudeep 

Does the test-results-reporter support html report?
I'm not get the desired results from a Junit(xml)report so want to know if it works with html report type to publish the results 

TIA

**Comment by ASaiAnudeep** [comment]
Not yet. We have an issue to track the work #38

**Reply by sgolyala1** [reply]
Great.Thanks
